staging 2025-12-10

Leandro13Silva13 opened the pull request:
Some body

mebo4b commented on src/main/java/de/diakonie/onlineberatung/log/LoggingEventListenerProvider.java:
```diff
@@ -0,0 +1,168 @@
+
+    sb.append(", cookies=[");
+    boolean f = true;
+    for (Map.Entry<String, Cookie> e : headers.getCookies().entrySet()) {
```
Could we rename "e" to "cookie" please?

mebo4b commented on src/main/java/de/diakonie/onlineberatung/log/LoggingEventListenerProvider.java:
```diff
@@ -0,0 +1,168 @@
+    }
+
+    sb.append(", cookies=[");
+    boolean f = true;
```
Could we also please rename "f" to something like "isFirstEntry"?

tkuzynow committed:
chore: dockerfile fix

tkuzynow committed:
chore: fix CVEs

tkuzynow committed:
chore: fix CVEs

tkuzynow committed:
chore: fix build

tkuzynow committed:
chore: exclude snake yaml build

tkuzynow committed:
chore: fix library dependency for new version of keycloak

patric-dosch-vi committed:
chore: fix library dependency for new version of keycloak

Upgrade keycloak to 22 also requires a Java update to 17 and some adjustments in the API, see https://www.keycloak.org/docs/latest/upgrading/index.html#migrating-to-22-0-2

For example: Migration from Java EE to Jakarta EE, Deprecated methods from Data Providers and Models were removed

patric-dosch-vi committed:
chore: cleanup code

tkuzynow committed:
Merge pull request #69 from Onlineberatung/OB-4625-fix-cve

Ob 4625 fix CVE

Leandro13Silva13 committed:
Merge branch 'develop' into staging-2025-12-10
